File: src/checkers-client/src/models/review.ts
export default class Review {
    id: string;
    playerName: string;
    content: string;
    postedOn: string;

    constructor(id: string, playerName: string, content: string, postedOn: string) {
        this.id = id;
        this.playerName = playerName;
        this.content = content;
        this.postedOn = postedOn;
    }
}

File: src/checkers-api/Models/PrimitiveModels/Review.cs
namespace checkers_api.Models.PrimitiveModels;

public class Review
{
    public string Id { get; }
    public string Content { get; }
    public string? PlayerId { get; }
    public DateTime PostedOn { get; }

    public Review(string id, string? playerId, string content, DateTime postedOn)
    {
        Id = id;
        PlayerId = playerId;
        Content = content;
        PostedOn = postedOn;
    }
}